source/_components/binary_sensor.mqtt.markdown

```diff
@@ -27,32 +31,37 @@ binary_sensor:
 
 Configuration variables:
 
+- **name** (*Optional*): The name of the binary sensor. Default is `MQTT Binary Sensor`.
 - **state_topic** (*Required*): The MQTT topic subscribed to receive sensor values.
-- **name** (*Optional*): The name of the sensor. Default is 'MQTT Sensor'. 
-- **qos** (*Optional*): The maximum QoS level of the state topic. Default is 0.
-- **payload_on** (*Optional*): The payload that represents on state. Default is "ON".
-- **payload_off** (*Optional*): The payload that represents  state. Default is "OFF".
+- **payload_on** (*Optional*): The payload that represents the on state. Default is `ON`.
+- **payload_off** (*Optional*): The payload that represents the off state. Default is `OFF`.
+- **availability_topic** (*Optional*): The MQTT topic subscribed to receive birth and LWT messages from the MQTT device. If `availability_topic` is not defined, the binary sensor availability state will always be `available`. If `availability_topic` is defined, the binary sensor availability state will be `unavailable` by default.
+- **payload_available** (*Optional*): The payload that represents the online state. Default is `online`.
+- **payload_not_available** (*Optional*): The payload that represents the offline state. Default is `offline`.
+- **qos** (*Optional*): The maximum QoS level to be used when receiving and publishing messages. Default is `0`.
```

Comment:
The binary sensor implementation in home assistant doesn't publish messages, right? I guess this is a standard phrase, but it could be confusing like this.